Sources/Datadog/FeaturesIntegration/WebView/WebEventBridge.swift

```diff
@@ -39,9 +39,9 @@ internal class WebEventBridge {
         self.rumEventConsumer = rumEventConsumer
     }
 
-    func consume(_ message: Any) throws {
-        guard let message = message as? String else {
-            throw WebEventError.invalidMessage(description: String(describing: message))
+    func consume(_ anyMessage: Any) throws {
+        guard let message = anyMessage as? String else {
```

Comment:
I don't have the context in mind but we could also be more flexible by using `CustomStringConvertible`?

Reply:
Indeed, that wont work with objc types!